presto-product-tests/conf/docker/singlenode-postgresql/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+
+  postgresql:
```

Comment:
automation fails with: `Caused by: java.net.UnknownHostException: postgres
`

Reply:
It works fine locally. On Travis for some reason hostname comes after servicename, and a `hostname` field is ignored. This is weird. For now i'm just going to revert back the service name.